### PR TITLE
Updated the dtreeviz object to complice with dtreeviz v2.0.0

### PR DESCRIPTION
**Pull request by haizadtarik**
Updated the code in explainerdashboard/explainers.py to resolve the issue describe in #243 
Another approach to resolve is to update the code in dtreeviz package to make it backward compatible

### CODE REVIEW

**Comment by oegedijk** [comment]
Hi @haizadtarik 

thanks a lot for the quick PR. Looks like a relatively straightforward fix. 

But perhaps we could still detect dtreeviz v1 and use the old API? Otherwise we will have to put in a requirement dtreeviz>2

(and I rather keep requirements minimal in order not to break people's existing environments...)

**Comment by haizadtarik** [comment]
Hey @oegedijk,

I have updated the code based on @tlapusan suggestion.
The code now works with v2.0.0 and v1.4.1

**Comment by parrt** [comment]
> Another approach to resolve is to update the code in dtreeviz package to make it backward compatible

We were shooting for that but I borked it up by reorging the code to clean up a massive trees.py file.

File: explainerdashboard/explainers.py [hunk 1]
```diff
@@ -25,8 +25,8 @@
 
 import shap
 
-import dtreeviz
-from dtreeviz.models.shadow_decision_tree import ShadowDecTree
+from dtreeviz import *
+from dtreeviz.trees import ShadowDecTree
```
Comment (parrt) [inline explainerdashboard/explainers.py:29]: Does this work in both 1.4.1 and 2.0.0?  Maybe this is safer and more specific?

```
from dtreeviz.models.shadow_decision_tree import ShadowDecTree
```

Comment (oegedijk) [reply explainerdashboard/explainers.py:29]: @haizadtarik did you have a chance to test the imports both with dtreeviz 1.4.1 and 2.0?

Comment (haizadtarik) [reply explainerdashboard/explainers.py:29]: Yes I tested to run the sample code from [README](https://github.com/oegedijk/explainerdashboard#examples-of-use) on dtreeviz v2.0.0 and dtreeviz v1.4.1 and did not encounter the import error and managed to bring up the dashboard ui

Using @parrt suggestion also worked on both version.

Let me know which is the best way to import the `ShadowDecTree` and I'll update this PR

Comment (oegedijk) [reply explainerdashboard/explainers.py:29]: Let's go with @parrt 's suggestion then:

```
from dtreeviz.models.shadow_decision_tree import ShadowDecTree
```

**Comment by oegedijk** [comment]
Cool, thanks so much @haizadtarik 

Will see if I can still get a release out before the year is over :)

**Comment by parrt** [comment]
Heads up, we have a 2.1 coming with lots of visual clean up and improvements, such as horizontal bar leaves not pie.

**Pull request merged**